### ISSUE TITLE
Feature Request: Automated paging jump

### ISSUE THREAD

**Issue by Photonut21** [chtaylo2/Roku-GooglePhotos#347]
I use this tv for my parents to only view photos of grandkids. Its not a big deal to restart the slide show but it is for them, also can i get a setting to auto jump to the next 1000 photos in the album. They have 11 kids and lots and lots of grand kids. I just need something that will show them photos 24/7. They are old it just needs to be easy for them. Google adds a platform for kids and grand kids to add photos.
Love the app this would just make my, and their life a little easier.

Thank you.

**Comment by Tellerik** [comment]
I agree with the 24/7 screen saver, my mother is in a elder care center and we can't visit her, however we wanted to display photos from all the family and grand kids. She has no idea how to reset or put on the slideshow, nor do the nurses have any idea to do a reset.  Any way to put this into the setting would be great.  I can't emphasize how important this is to our family.  BTW, I was so impressed with your app, I donated a few months ago, keep up the good work.

**Comment by chtaylo2** [comment]
@Tellerik - Thank you for the donation, it's very much appreciated. I've put together an additional FAQ on the timeout and why it's imposed. Please see: http://www.photoviewapp.com/faq/#timeout

What I would recommend in both of your cases is to set a screensaver timeout of 1-5 minutes with PhotoView as the selected screensaver. Then, without any user action, it'll load up the screensaver pulling in the latest photos from whatever album you select. In the future maybe we can change how this works. 

@Photonut21 - I'll keep this story open as a feature request on the auto-jump. Not sure how we'd add this into the standard flow, but will think about it.
